modules/cog-fdo-shell.c

```diff
@@ -30,6 +44,322 @@ typedef struct {
     CogShell parent;
 } CogFdoShell;
 
+struct wpe_input_touch_event_raw touch_points[10];
+
+extern PwlData wl_data;
+extern PwlEGLData egl_data;
+extern PwlWinData win_data;
+extern PwlXKBData xkb_data;
+
+static struct {
+    struct wpe_view_backend_exportable_fdo *exportable;
+} wpe_host_data;
+
+static struct {
```

Comment:
This information must to be self contained now in a CogView. E.x: each CogView should a particular Backend (for isolation)